features/adults/conviction_military.feature

```diff
@@ -9,19 +9,20 @@ Feature: Conviction
     Then I should see "<known_date_header>"
 
     And I enter a valid date
-    Then I should see "<length_type_header>"
```

Comment:
I don't understand this. Why remove the variables?

Reply:
I removed the variables because it felt that the repetition is on the table instead of the setup. 
The values are all the same, so I thought it would be less repetitive if we just add it straight into the scenario.

Reply:
After updating the title, now it makes more sense to have the variables